app/entry.server.tsx

```diff
@@ -1,6 +1,13 @@
 import type { EntryContext } from "@remix-run/node";
 import { RemixServer } from "@remix-run/react";
 import { renderToString } from "react-dom/server";
+import { fetchAllGuildMembers } from "./services/discord.server";
+import redisClient from "./services/redis.server";
+
+setInterval(async () => {
```

Comment:
The cache population here should ideally be decoupled from the main web server app, otherwise each instance of the app will also be populating the cache when only 1 needs to, leading to wasted compute and discord api requests.

A job or long running app deployed with 1 instance to handle cache population would work better here.

Reply:
Will do this in a follow-up PR, need to work out where I'll host it, for now, this one shouldn't cook the limit so can work

The issue with scheduled machines https://community.fly.io/t/new-feature-scheduled-machines/7398 is it'll run at most hourly, so I'm not sure if it's viable. I need to go deeper on how to decouple this.

Reply:
That makes sense! we can eat the cost for the moment